src/components/Tool/FixSendNewsletter/FixSendNewsletter.tsx

```diff
@@ -39,101 +45,126 @@ const useStyles = makeStyles(() => ({
     display: 'flex',
     justifyContent: 'center',
   },
-}));
-
-const testData = [
-  {
-    title: 'Test Contact and Friends',
-    name: 'Test Contact',
-    tag: 'Partner - Financial',
-    address: {
-      street: '70 Test Ave',
-      city: 'Vancouver BC V5Z 2V7',
-    },
-    source: 'Source: DonorHub (12/16/2014)',
-    newsletterType: 'physical',
-    email: 'test@test.com',
+  buttonBlue: {
+    backgroundColor: theme.palette.mpdxBlue.main,
+    marginTop: theme.spacing(1),
+    color: 'white',
   },
-  {
-    title: 'Anonymous Test',
-    newsletterType: 'both',
+  buttonIcon: {
+    marginRight: theme.spacing(1),
   },
-];
+}));
 
 const FixSendNewsletter = (): ReactElement => {
   const classes = useStyles();
-  const [test, setTest] = useState(testData);
   const { t } = useTranslation();
+  const accountListId = useAccountListId();
+  const { data, loading } = useGetInvalidNewsletterQuery({
+    variables: { accountListId: accountListId || '' },
+  });
 
-  const toggleData = (): void => {
-    test.length > 0 ? setTest([]) : setTest(testData);
-  };
+  const filtered = data?.contacts.nodes.filter(
+    (el) =>
+      el.status === 'PARTNER_FINANCIAL' ||
+      el.status === 'PARTNER_SPECIAL' ||
+      el.status === 'PARTNER_PRAY',
+  );
```

Comment:
not sure if there is a better way to do this. from my understanding there isn't currently a way to do this directly onto the gql query

Reply:
I think you send a bunch of statuses:
```diff
-  contactsFilter: { newsletter: NO_VALUE }
+  contactsFilter: { newsletter: NO_VALUE, status: [PARTNER_FINANCIAL, PARTNER_SPECIAL, PARTNER_PRAY] }